server.js

```diff
@@ -23,6 +28,22 @@ app.post("/api/upload", async (request, response) => {
   }
 });
 
+// GET-Route from MongoDB
+app.get("/api/sven/:tags", async (request, response) => {
+  const { tags } = request.params;
+  try {
+    const entryValue = await getData(tags);
```

Comment:
```suggestion
    const entryValue = await getImage(tags);
```
What kind of data do you get

Reply:
👍
Thanks for reviewing!